common/fdlimit/fdlimit_test.go

```diff
@@ -24,7 +24,7 @@ import (
 // TestFileDescriptorLimits simply tests whether the file descriptor allowance
 // per this process can be retrieved.
 func TestFileDescriptorLimits(t *testing.T) {
-	target := 4096
+	target := 20000
```

Comment:
Could you tell me why this fd limit in test needs to be increased?

Reply:
In the situation we faced the problem, the limit was increased from 2048 to 4096, but wasn't increase from 4096 to 10096. Please check the below logs =)

```
WARN[09/22,16:40:40 +09] [39] Increase the file descriptor limit of the process for RPC servers  oldLimit=2048 newLimit=8048
INFO[09/22,16:40:40 +09] [44] Start listening chain head event to update stakingInfoCache.
WARN[09/22,16:40:40 +09] [39] Increase the file descriptor limit of the process for RPC servers  oldLimit=4096 newLimit=10096
INFO[09/22,16:40:40 +09] [40] IPC endpoint opened                       url=/data/kend/data/klay.ipc
WARN[09/22,16:40:40 +09] [39] Increase the file descriptor limit of the process for RPC servers  oldLimit=4096 newLimit=10096
INFO[09/22,16:40:40 +09] [40] FastHTTP endpoint opened                  url=http://0.0.0.0:8551      cors=* vhosts=*
WARN[09/22,16:40:40 +09] [39] Increase the file descriptor limit of the process for RPC servers  oldLimit=4096 newLimit=10096
INFO[09/22,16:40:40 +09] [40] FastWebSocket endpoint opened             url=ws://0.0.0.0:8552
```